### PR TITLE
Benchmark

### PR DESCRIPTION
**Pull request by Shugaba-Wuta**
#260  Performance benchmark of MLFlow and FastTrackML

### CODE REVIEW

**Comment by suprjinx** [comment]
I wouldn't expect any file changes outside of the `/benchmarks` folder -- looks like some other changes got caught up here

File: .gitignore
Comment (Shugaba-Wuta) [inline .gitignore:1]: @suprjinx should the files generated benchmark_outputs be tracked by Git? I am thinking of leaving `benchmark_outputs**`

Comment (suprjinx) [reply .gitignore:1]: good question -- i don't think we need to track right now

**Review by suprjinx — APPROVED** [review]
looks good!

**Pull request merged**